src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/ProcessTest.cs

```diff
@@ -3,580 +3,381 @@
 
 using System;
 using System.Diagnostics;
-
+using System.Runtime.InteropServices;
 using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.Win32.SafeHandles;
 using Xunit;
 using System.Text;
 
 namespace System.Diagnostics.ProcessTests
 {
-    public partial class ProcessTest
+    public partial class ProcessTest : IDisposable
     {
-        static ProcessTest()
-        {
-            TryKillExistingTestProcesses();
-        }
 
-        private static void TryKillExistingTestProcesses()
+        private const string s_ProcessName = "ProcessTest_ConsoleApp.exe";
+        private static bool s_Process_EnableRaiseEvents_isExitedEventHandlerCalled = false;
+        private Process s_process;
+
+        public ProcessTest()
         {
-            try
-            {
-                foreach (Process p in Process.GetProcessesByName(s_ProcessName.Replace(".exe", "")))
-                {
-                    p.Kill();
-                }
-            }
-            catch { }; // do nothing 
+            s_process = CreateProcessInfinite();
```

Comment:
CreateProcessInfinite() is explicitly killed much before that. Personally I liked Thread.Sleep(-1) and ProcessTest.Dispose() method combination however this was explicitly replaced at one point to be 30 secs so I let it stay.